packages/webview_flutter/README.md

```diff
@@ -0,0 +1,52 @@
+# webview_flutter_tizen
+
+The Tizen implementation of [`webview_flutter`](https://github.com/flutter/plugins/tree/master/packages/webview_flutter).
+
+## Supported devices
+
+This plugin is available on these types of devices:
+
+- Galaxy Watch or TV (running Tizen 5.5 or later)
+
+## Usage
+
+```yaml
+dependencies:
+  webview_flutter: ^1.0.6
+  webview_flutter_tizen: ^1.0.0
+```
+
+To enable tizen implementation, set `WebView.platform = TizenWebView();` in `initState()`.
+For example:
+
+```dart
+import 'dart:io';
+
+import 'package:webview_flutter/webview_flutter.dart';
+import 'package:webview_flutter_tizen/webview_flutter_tizen.dart';
+
+class WebViewExample extends StatefulWidget {
+  @override
+  WebViewExampleState createState() => WebViewExampleState();
+}
+
+class WebViewExampleState extends State<WebViewExample> {
+  @override
+  void initState() {
+    super.initState();
+    WebView.platform = TizenWebView();
```

Comment:
I'm sorry but I don't understand what you said. Please let me know what I should do.

Reply:
Flutter is a cross-platform framework, and the same app code is shared across different platforms. So if you write code like this:
```dart
WebView.platform = TizenWebView();
```
this app will break on platforms other than Tizen, so you need to check which platform the app is currently running on:
```dart
if (Platform.environment.containsKey('TIZEN_API_VERSION')) { // equivalent of `Platform.isAndroid` on Android
  WebView.platform = TizenWebView();
}
```
However this is quite cumbersome for app developers to do, so we can let the plugin itself do its initialization during app startup. Specify `dartPluginClass` as follows in puspec.yaml (it could be anything other than `TizenWebView`):

```yaml
tizen:
  pluginClass: WebviewFlutterTizenPlugin
  fileName: webview_flutter_tizen_plugin.h
  dartPluginClass: TizenWebView
```

and create a static register method in `TizenWebView`:

```dart
class TizenWebView implements WebViewPlatform {
  /// Registers this class as the default instance of [WebViewPlatform].
  static void register() {
    WebView.platform = TizenWebView();
  }
  ...
```

That's it. You can refer to other Dart plugins such as url_launcher_tizen and shared_preferences_tizen which do the similar thing.

Reply:
I merged https://github.com/flutter-tizen/flutter-tizen/pull/65 so you can test the above code now.

Reply:
I updated it. Thanks.